extensions/wikia/FeedsAndPosts/FeedsAndPostsHooks.class.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class FeedsAndPostsHooks {
+
+	const WIKI_IDS = [
+		147, // Star Wars/Wookiepedia
+		509, // harry Potter
+		1706, // The Elder Scrolls
+		1619010, // xkxd (for testing purposes)
+	];
```

Comment:
probably more sustainable to create WF var that's default false to act as gatekeeper to enable/disable the whole extension, and enable it on wikis as needed.